influxdb/client.py

```diff
@@ -53,8 +53,11 @@ class InfluxDBClient(object):
     :param timeout: number of seconds Requests will wait for your client to
         establish a connection, defaults to None
     :type timeout: int
-    :param retries: number of retries your client will try before aborting,
+    :param retries: how many times your client will try before aborting,
         defaults to 3. 0 indicates try until success
+        For example:
+        1 - try only once (without retry),
+        2 - maximum two tries (including one retry)
```

Comment:
Can you please also add a line here for the default option?

```
:param retries: number of attempts your client will make before aborting, defaults to 3
0 - try until success
1 - attempt only once (without retry)
2 - maximum two attempts (including one retry)
3 - maximum three attempts (default option)
```

Reply:
done